packages/-relationship-performance-test-app/app/routes/application.js

```diff
@@ -0,0 +1,22 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
```

Comment:
having this be `application` route is fine for now, but we should separate out various perf scenarios into different routes going forward, for instance so that we can create a second similar scenario without the inverse relationships being defined.